test/system/import.test.ts

```diff
@@ -42,9 +42,7 @@ describe('`snyk-api-import import`', () => {
         }
         expect(stderr).toEqual('');
         expect(err).toBeNull();
-        expect(stdout.trim()).toMatch(`project(s)
-Processed 1 out of a total of 1 targets
-Check the logs for any failures located at:`);
+        expect(stdout.trim()).toMatch(`Check the logs for any failures located at:`);
```

Comment:
why did this change? Now we can't see if any targets were processed

Reply:
it doesn't but the test fails, there must be some spaces or special characters. I couldn't find it. I understand what you say. I'll see if I can find something

Reply:
try to do `.Equals()` locally that will show you the diff